### PR TITLE
jeti added to repository

### PR DESCRIPTION
**Pull request by mrshampoo**
just added jeti 1.1 :)

### CODE REVIEW

**Comment by Vaelatern** [comment]
Please read the end of the manual to see how your commit should be formatted.
There should be no merge commits in the PR.

**Comment by mrshampoo** [comment]
oh ok.

File: srcpkgs/jeti/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'jeti'
+pkgname=jeti
+version=1.1
+revision=1
+wrksrc=jeti-filemanager-master
+homepage="https://github.com/mrshampoo/jeti-filemanager/"
+makedepends="ncurses-devel"
+short_desc="filemanager,(A Totoal commander clone, in ncruses)"
```
Comment (Vaelatern) [inline srcpkgs/jeti/template]: Please write this more cleanly. Perhaps `A Total Commander filemanager ncurses clone`

File: srcpkgs/jeti/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+maintainer="Harri Leino <mr.leino@gmail.com>"
+license="GPL"
+distfiles="https://github.com/mrshampoo/jeti-filemanager/archive/master.tar.gz>jeti-filemanager-master.tar.gz"
+checksum=13c5e97a2108b0e54fc026e39678a9da74a832135624af03e00f2c4e61fa12ce
+
+do_build(){
```
Comment (Vaelatern) [inline srcpkgs/jeti/template]: `build_style=gnu-makefile` instead of these functions.

File: srcpkgs/jeti/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+pkgname=jeti
+version=1.1
+revision=1
+wrksrc=jeti-filemanager-master
+homepage="https://github.com/mrshampoo/jeti-filemanager/"
+makedepends="ncurses-devel"
+short_desc="filemanager,(A Totoal commander clone, in ncruses)"
+maintainer="Harri Leino <mr.leino@gmail.com>"
+license="GPL"
+distfiles="https://github.com/mrshampoo/jeti-filemanager/archive/master.tar.gz>jeti-filemanager-master.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/jeti/template]: There is no tagged release of this software and it is being actively developed. Please get a tagged release and download that instead.

**Comment by mrshampoo** [comment]
humm ok...

**Comment by mrshampoo** [comment]
how do i tag my software int github?

**Comment by Vaelatern** [comment]
You tag a commit. You probably want to tag it "1.1" or something.

**Comment by mrshampoo** [comment]
as in?
git commit -m "1.1"

**Comment by Vaelatern** [comment]
http://git-scm.com/book/en/v2/Git-Basics-Tagging

**Comment by mrshampoo** [comment]
ok done. now there is a 1.1 at https://github.com/mrshampoo/jeti-filemanager/archive/1.1.tar.gz
but will this change the md5sum?

**Comment by Vaelatern** [comment]
Find out.

**Comment by mrshampoo** [comment]
fuck! it did, feels like doing everything all over bbl

**Comment by mrshampoo** [comment]
Try number 2: aaand action!

**Comment by mrshampoo** [comment]
i did not know what you meant by 'PR' unfortunately :(

**Comment by TheDoctorsLife** [comment]
PR is short for Pull Request.

**Comment by mrshampoo** [comment]
humm okej?. .

**Comment by mrshampoo** [comment]
hope i did not do that then :p

**Comment by mrshampoo** [comment]
how do I know if my template got accepted?

**Comment by Vaelatern** [comment]
It didn't yet. You'll know when it happens.

**Comment by mrshampoo** [comment]
ah ok :)

**Comment by Vaelatern** [comment]
There should be only 1 commit in this pull request and it should have the proper naming.

**Comment by mrshampoo** [comment]
how can i uncommit?

File: srcpkgs/jeti/template [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'jeti'
+pkgname=jeti
+version=1.1
+revision=2
+wrksrc=jeti-filemanager-$version
+build_style=gnu-makefile
+homepage="https://github.com/mrshampoo/jeti-filemanager/"
+makedepends="ncurses-devel"
+short_desc="A Totoal commander filemanager ncurses clone"
```
Comment (Vaelatern) [inline srcpkgs/jeti/template]: You sure this isn't `Total Commander`

File: srcpkgs/jeti/template [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'jeti'
+pkgname=jeti
+version=1.1
+revision=2
```
Comment (Vaelatern) [inline srcpkgs/jeti/template]: Revision 1

File: srcpkgs/jeti/template [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+wrksrc=jeti-filemanager-$version
+build_style=gnu-makefile
+homepage="https://github.com/mrshampoo/jeti-filemanager/"
+makedepends="ncurses-devel"
+short_desc="A Totoal commander filemanager ncurses clone"
+maintainer="Harri Leino <mr.leino@gmail.com>"
+license="GPL"
+distfiles="https://github.com/mrshampoo/jeti-filemanager/archive/$version.tar.gz>jeti-filemanager-$version.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/jeti/template]: `distfiles="https://github.com/mrshampoo/jeti-filemanager/archive/${version}.tar.gz"

File: srcpkgs/jeti/template [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+pkgname=jeti
+version=1.1
+revision=2
+wrksrc=jeti-filemanager-$version
+build_style=gnu-makefile
+homepage="https://github.com/mrshampoo/jeti-filemanager/"
+makedepends="ncurses-devel"
+short_desc="A Totoal commander filemanager ncurses clone"
+maintainer="Harri Leino <mr.leino@gmail.com>"
+license="GPL"
```
Comment (Vaelatern) [inline srcpkgs/jeti/template]: `GPL-3`

File: srcpkgs/jeti/template [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'jeti'
+pkgname=jeti
+version=1.1
+revision=2
+wrksrc=jeti-filemanager-$version
```
Comment (Vaelatern) [inline srcpkgs/jeti/template]: You probably should name the package `jeti-filemanager` to help keep naming consistent with what the project is called.

**Comment by mrshampoo** [comment]
Ok I give up. I need some sleep.

**Comment by Vaelatern** [comment]
You probably want to try `git rebase -i HEAD~7` and learn what to do, probably something involving squashing.

**Pull request closed without merging**